src/corehost/corehost.cpp

```diff
@@ -151,6 +146,12 @@ int run(const int argc, const pal::char_t* argv[])
         trace::error(_X("A fatal error was encountered. This executable was not bound to load a managed DLL."));
         return StatusCode::AppHostExeNotBoundFailure;
     }
+#else
+	if (argc <= 1)
+	{
+		trace::println(_X("Use dotnet --help to get help with using the correct arguments"));
```

Comment:
This actually cuts out a completely valid scenario.  Today, even if the `sdk` is installed, I can invoke `dotnet` with no args, and I can get the host's help.  With this change, there is no way to get the host's help message if the `sdk` is installed.